docs/getting-started-guides/coreos/bare_metal_calico.md

```diff
@@ -95,7 +125,7 @@ Once complete, eject the bootable ISO and restart the server.  When it comes bac
 
 First, boot up the node machine using the bootable ISO we downloaded earlier.  You should be automatically logged in as the `core` user.
 
-Make a copy of the `node-config-template.yaml` in the `calico-kubernetes` repository for this machine.
+Make a copy of the `node-config-template.yaml` for this machine.
```

Comment:
I think this and the other template from above should be obtained directly with wget, e.g. 
`wget -O master-config.yaml https://raw.githubusercontent.com/projectcalico/calico-kubernetes/master/config/cloud-config/master-config-template.yaml`